cumulusci/tasks/salesforce/sourcetracking.py

```diff
@@ -302,10 +302,11 @@ def retrieve_components(
                     os.path.join(package_xml_path, "package.xml"),
                     "-w",
                     "5",
+                    "-c",
```

Comment:
`(cci_venv) l.ramireddy@lramire-ltmnypt easy-org-provision % cci task run retrieve_changes --org dev
[09/24/24 11:17:35] Getting org info from Salesforce CLI for test-dimuhhn7ik7c@example.com                             
[09/24/24 11:17:39] Beginning task: RetrieveChanges                                                                    
                    As user: test-dimuhhn7ik7c@example.com                                                             
                    In org: 00DO4000006Oxhp                                                                            
                                                                                                                       
                    Querying Salesforce for changed source members                                                     
                    Profile: Admin                                                                                     
[09/24/24 11:17:40] Retrieving components with command: sf project retrieve start -a 59.0 -x                           
                    /var/folders/tq/td7hb8n97tzdkw6011gqn55h0000gn/T/tmp35ank3o5/package.xml -w 5                      
 STATE    FULL NAME TYPE    FILE PATH                                                                                                          
 ──────── ───────── ─────── ────────────────────────────────────────────────────────────────────────────────────────────────────────────────── 
 Conflict Admin     Profile /Users/l.ramireddy/Desktop/easy-org-provision/force-app/main/default/provision_org/profiles/Admin.profile-meta.xml 
Preparing retrieve request... Error
Error (1): There are changes in your local files that conflict with the org changes you're trying to retrieve.

Try this:

To overwrite the local changes, rerun this command with the --ignore-conflicts flag.
To overwrite the remote changes, run the "sf project deploy start" command with the --ignore-conflicts flag.


Error: Command exited with return code 1`

Which used to work properly with the old sfdx commands. To avoid this errors added the --ignore-conflicts